app/api/v1/endpoints/upload.py

```diff
@@ -0,0 +1,40 @@
+from fastapi import APIRouter, HTTPException, File, UploadFile
+from pydantic import BaseModel
+from azure.storage.blob import BlobServiceClient
+import os
+
+# APIRouter 생성
+router = APIRouter()
+
+# Azure Blob Storage 설정
+# 애저 커넥팅
+# 컨테이너 이름
+
+class ImageURL(BaseModel):
+    url: str
+
+@router.post("/")
```

Comment:
그리고 지금 schema를 안만드셔서 swagger.yaml response 타입이 string으로 나오고 있거든요 schema 만드셔서 아래처럼 추가해주세요
```suggestion
@router.post("/", response_model=upload_schema.Upload)
```